### DISCUSSION TITLE
Here are some presets that I made,

### DISCUSSION THREAD

**Discussion by rekkol** [ntsc-rs/ntsc-rs#158] [Presets]
I haven't posted them before because I'm always Tweaking them, But now I think i'm Good With them now
(Update: I was reading @Degamisu's #140 and changed the Early Color TV, U-Matic and VHS's Bandwidth scale to 1.5)

Here are some photos
![Composite Video](https://github.com/user-attachments/assets/7a3af17b-4f14-4874-a67a-26e2aea815f3)
Composite Video With Flashy Rainbow effects
![Early BW TV](https://github.com/user-attachments/assets/f1f6f37e-61c1-405b-b63f-80bce38de5f3)
Early Black and White TV Look from the 50s -60s
![Early Color TV](https://github.com/user-attachments/assets/0dc13798-799a-4728-a3ba-5b15b259f38c)
Early color TV Look from the 60s -70s
![U-Matic](https://github.com/user-attachments/assets/4e02b842-02cb-4e90-a008-50c843f66657)
U-Matic Videotape from the 70s-80s
![VHS](https://github.com/user-attachments/assets/54e85ee6-3ad2-41c5-b177-0c805a4b8c2e)
VHS Video from the 80s-2000s 

[Rekcol's NTSC Presets.zip](https://github.com/user-attachments/files/16614508/Rekcol.s.NTSC.Presets.zip)

Also heres the old ones in case u want them
[(OG) Rekcol's NTSC Presets.zip](https://github.com/user-attachments/files/16614542/OG.Rekcol.s.NTSC.Presets.zip)

**Comment by Degamisu** [comment]
Well done @rekkol! My favorite is the B&W TV. Very accurate!

**Comment by Henrycounty** [comment]
Those presets look awesome!

**Comment by akuncadangans1tu-beep** [comment]
Damn!, realistic preset!, i like it!

**Reply by walkerpatricia706-spec** [reply]
I like these cool presets.
